src/main/resources/assets/admin/common/js/ui/treegrid/TreeGrid.ts

```diff
@@ -1198,13 +1011,13 @@ export class TreeGrid<DATA>
     private initKeyBindings() {
         if (!this.gridOptions.isMultipleSelectionDisabled()) {
             this.keyBindings = [
-                new KeyBinding('shift+up', (event: ExtendedKeyboardEvent) => {
+                new KeyBinding('shift+up', (event: Mousetrap.ExtendedKeyboardEvent) => {
```

Comment:
Codacy found an issue: ['Mousetrap' is not defined.](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6422415)